cylc/flow/scripts/trigger.py

```diff
@@ -17,11 +17,17 @@
 
 """cylc trigger [OPTIONS] ARGS
 
-Force tasks to run regardless of prerequisites.
+Manually trigger tasks regardless of prerequisites, even in a paused workflow.
 
-* Triggering an unqueued waiting task queues it, regardless of prerequisites.
-* Triggering a queued task submits it, regardless of queue limiting.
-* Triggering an active task has no effect (it already triggered).
+Triggering an unqueued task queues it, to run when released by the queue.
+Triggering a queued task runs it immediately regardless of queue limiting.
+So you may need to trigger tasks twice if queue limiting is in effect.
+
+If the workflow is paused queued waiting tasks will not run (unless manually
+triggered) until the workflow is resumed, even if the queue empties out. This
+includes tasks queued by manual triggering, when queue limits are in effect.
```

Comment:
```suggestion
Triggering a queued task runs it immediately.

Triggering a task that is not yet queued (i.e. "queue-limited") will queue it;
it will run when released from the queue. So you may need to trigger tasks
twice if queue limiting is in effect.
```

Reply:
As for the CLI help:
```suggestion
Triggering a task that is not yet queued, queues it.

Triggering a queued task runs it immediately.
                        
Queues release tasks to run when their active task count drops below the queue
limit. So, depending on activity, you may need to trigger a task twice to make
it run immediately. 
```

Reply:
@MetRonnie - I've tweaked this for consistency with your suggestion for the CLI help, so will merge it.